src/development/platform-integration/android/platform-views.md

```diff
@@ -183,7 +183,6 @@ package dev.flutter.example
 
 import android.content.Context
```

Comment:
@khanhnwin can this be turned into a snippet somehow? I dislike code in docs that isn't tested somehow.